code/modules/jobs/access.dm

```diff
@@ -31,6 +31,8 @@
 		var/mob/living/simple_animal/animal = accessor
 		if(check_access(animal.access_card))
 			return TRUE
+	else if(isbrain(accessor) && istype(accessor.loc, /obj/item/mmi))
+		return TRUE //Like with AI, MMI can also do whatever if they... you know, grow up and get metal forklift arms and a flamethrower or something. (mecha)
```

Comment:
I feel like this could be flawed, as someone could place a brain into an mmi and throw them at doors to open the door. Can you test to make sure this isn't actually possible?

Also, I think MMIs shouldn't have default access to everything. The fact that this is limited to borgs is because they're usually linked to an AI and are under a lawset, unlike MMIs which just have the goal to 'serve their master.'

I think mechs can already be configured to be given access to doors by tweaking their maintenance settings and controlling what accesses are required to ride the mech.